packages/flutter_tools/lib/src/web/compile.dart

```diff
@@ -135,18 +135,22 @@ class WebBuilder {
 /// Web rendering backend mode.
 enum WebRendererMode {
   /// Auto detects which rendering backend to use.
-  autoDetect,
+  auto(helpText: 'Use the HTML renderer on mobile devices, and CanvasKit on desktop devices.'),
   /// Always uses canvaskit.
-  canvaskit,
+  canvaskit(helpText: 'Always use the CanvasKit renderer. This renderer uses WebGL and WebAssembly to render graphics.'),
   /// Always uses html.
-  html,
+  html(helpText: 'Always use the HTML renderer. This renderer uses a combination of HTML, CSS, SVG, 2D Canvas, and WebGL.'),
   /// Always use skwasm.
-  skwasm,
+  skwasm(helpText: 'Always use the experimental skwasm renderer.');
+
+  const WebRendererMode({required this.helpText});
+
+  final String helpText;
```

Comment:
Why is `helpText` the property that defines the enum?

Can `helpText` be a getter, similar to what you did with `get dartDefines`, and just use the free `name` property as the value of the enum (maybe also a .toString() method that returns `.name` so you maybe get free Enum -> String conversions?)

Reply:
(PS: I might be changing my name, and maybe making the enum hold more properties, rather than having methods that `switch(this)` :P)